pkg/config/config_template.yaml

```diff
@@ -899,10 +899,10 @@ api_key:
   ## Policy files
   # policies:
 
-    ## @param dir - string - folder from where the policies will be loaded
+    ## @param dir - string - default: /etc/datadog-agent/runtime-security.d
     ## Path from where the policy files will be loaded
     #
-    dir: /etc/datadog-agent/runtime_security_agent.d
+    # dir: /etc/datadog-agent/runtime-security.d
```

Comment:
When this dir is not initialized - system probe currently fails to start. Do we create this dir in the docker image?

Reply:
We added the folder creation during the package creation